test/support/test_config.h

```diff
@@ -120,6 +120,54 @@
 #define TEST_HAS_NO_INT128
 #define _PSTL_TEST_COMPLEX_NON_FLOAT_AVAILABLE (_MSVC_STL_VERSION < 143)
 
+#define _PSTL_TEST_COMPLEX_OP_BROKEN (_MSVC_STL_VERSION && _MSVC_STL_VERSION <= 143)

```

Comment:
is the reason why it does not work well understood?

Reply:
As far as I seen these errors has place due MS standard library implementation.